docker/README.md

```diff
@@ -141,6 +141,21 @@ ENV Vars
   * used to only run a subset of tests
   * see .travis.yml for exact options
 * REUSE_DB
-  * Same as normal REUSE_DB
- 
+  * Same as normal `REUSE_DB`
+* DOCKER_HQ_OVERLAY=[**none**|overlayfs|**aufs**]
```

Comment:
I assume bold indicates the default. Are there two defaults for this one?

Reply:
Yes, one for travis, one for otherwise (that's how the logic is currently written).